UnitsNet/Comparison.cs

```diff
@@ -26,14 +26,23 @@ namespace UnitsNet
     public static class Comparison
     {
         /// <summary>
+        ///     <para>
         ///     Checks if two values are equal with a given relative or absolute tolerance.
-        ///     Relative tolerance is when the difference between the two values is not greater than referenceValue * tolerance.
-        ///     Absolute tolerance is when the absolute difference between the two values is not greater than the tolerance value.
+        ///     </para>
+        ///     <para>
+        ///     Relative tolerance is defined as the maximum allowable absolute difference between <paramref name="referenceValue"/> and
+        ///     <paramref name="otherValue"/> as a percentage of <paramref name="referenceValue"/>. For example, a relative tolerance
+        ///     of 0.01 means the difference must be within +/- 1% of <paramref name="referenceValue"/> to be considered equal.
+        ///     </para>
+        ///     <para>
+        ///     Absolute tolerance is defined as the maximum allowable absolute difference between <paramref name="referenceValue"/> and
+        ///     <paramref name="otherValue"/> as a fixed number.
```

Comment:
Fixed number, but in what unit?

Reply:
In this case it's just a double, so there is no unit. But when you're looking at the Equals on a quantity, the reference value is "this" so it's a fixed number of "this" unit.

```csharp
var a = Length.FromMeters( 100.0 );
var b = Length.FromInches( 5.0 );
a.Equals( b, 0.001 )
```

In this case it's absolute of a. So within 0.001 meters.

Reply:
Okay sure, that is one way to look at it. I will still argue that there is a unit in play here and it is the constructed unit of `a`, which is the most sensible choice by all means! However, I would have to read the current xmldoc more than once to understand what will actually happen, since there is conversions going on here and you are naturally not comparing the values 100 and 5 head on. If the xmldoc rather said the tolerance was in the `referenceValue`'s `Unit` then at least in my eyes it would be much more intuitive. I guess we just have different perspective on this and read it a bit differently.

So for me, a slight change in wording of xmldoc would make it a lot more intuitive.

Reply:
I think changing the wording definitely makes sense regardless!